hub/core/chunk_engine/write.py

```diff
@@ -35,16 +87,20 @@ def write_bytes(
     #  for performance
     last_chunk_name, last_chunk = _get_last_chunk(key, storage, index_meta)
 
+    # refactor TODO: move to separate function
```

Comment:
Reminder

Reply:
this TODO is for later, its an unnecessary refactor for now

Reply:
i really narrowed in on what needed to be refactored